js/speaker.js

```diff
@@ -0,0 +1,91 @@
+const speakersData = [
+  {
+    id: 1,
+    name: 'Oprah Winfrey',
+    photo: 'images/speakers/oprah.jpeg',
+    description: 'CEO OWN and Philanthropist',
+    about:
+      'Oprah Winfrey famously Business Owner and philanthropy over the years',
+  },
+  {
+    id: 2,
+    name: 'Amaka Osakwe',
+    photo: 'images/speakers/amaka-osakwe.jpg',
+    description: 'Enterpreneur and Fashion Designer',
+    about:
+      'Amaka Osakwe is a Nigerian fashion designer and creator of the African-based fashion label named Maki Oh.',
+  },
+  {
+    id: 3,
+    name: 'Linda Ikeji',
+    photo: 'images/speakers/Linda.png',
+    description: 'CEO LindaIkejia Media',
+    about: 'Linda owns one of the fastest growing Media company in Nigeria',
+  },
+  {
+    id: 4,
+    name: 'Luvvie Ajayi',
+    photo: 'images/speakers/luvvie.jpeg',
+    description: 'Writer ',
+    about:
+      'Changing the world by owning up to the truth and encouraging other to do so',
+  },
+  {
+    id: 5,
+    name: 'Shola Eweniyi',
+    photo: 'images/speakers/shola2.jpeg',
+    description: 'Co-founder and CEO of Paystack',
+    about:
+      'Shola is Nigerian software engineer and entrepreneur focused on building payment solutions',
+  },
+
+  {
+    id: 6,
+    name: 'Onomeh Onwuanibe',
+    photo: 'images/speakers/beyonce.jpeg',
+    description: 'Founder, GGC',
+    about: 'Software Engineer.',
+  },
+];
+
+const speakersBody = document.querySelector('.speaker-list');
+const seeMoreButton = document.querySelector('.see-more');
+// add speakers dynamically to page
+speakersData.forEach((speaker) => {
+  speakersBody.innerHTML += `
+      <div id = '${speaker.id}' class='speaker'>
+        <span class='speaker-img'>
+          <img src=${speaker.photo} alt='Paystack, CEO'>
+        </span>
```

Comment:
- [ ] Please use the same image size for your all speakers, Right now they are different sizes and not looking good. It will improve the design and make this project more professional. You can use this website [picresize.com](https://picresize.com/) to resize or crop the image.

### See the image for reference

![image](https://user-images.githubusercontent.com/83803414/158951277-c70b2a14-d8f2-425e-a605-de98f64cafb5.png)